.rubocop.yml

```diff
@@ -1,9 +1,19 @@
-require: rubocop-performance
-inherit_from:
-  - https://gitlab.com/bsm/misc/raw/master/rubocop/default.yml
+inherit_gem:
+  rubocop-bsm:
+    - default.yml
+inherit_mode:
+  merge:
+    - Exclude
 
 AllCops:
   TargetRubyVersion: "2.6"
 
 Metrics/ParameterLists:
   Max: 8
+
+RSpec/FilePath:
+  Exclude:
+    - core/spec/bfs/helpers_spec.rb
+
+RSpec/MessageSpies:
+  Enabled: false
```

Comment:
```
core/spec/bfs_spec.rb:13:25: C: RSpec/MessageSpies: Prefer have_received for setting message expectations. Setup bucket as a spy using allow or instance_spy.
      expect(bucket).to receive(:close)
```

I think suggested usage is a bit too verbose, but can change to what it suggests (allow and then expect... have_received)